tests/Error/ErrorTest.php

```diff
@@ -9,131 +12,104 @@
 
 class ErrorTest extends TestCase
 {
-    /**
-     * @it uses the stack of an original error
-     */
-    public function testUsesTheStackOfAnOriginalError()
```

Comment:
Please do not delete those comments in tests. They are very helpful when migrating tests from the reference implementation.

Reply:
Ayright, what is `@it` annotation's meaning?

Reply:
It contains the name of the mocha test case from the reference implementation. I.e. `it('uses the stack of an original error')`.

This way it is easy to find the same test case in the reference implementation to check if something had changed there.

Reply:
Can we use `* @see it('uses the stack of an original error')` instead?

Reply:
I don't care much as long as the original test description is preserved

Reply:
Cool, I'll use standard @see and we can drop confusing `@it` annotation ;)